### ISSUE TITLE
DynamicTraceEventParser does NOT support field type "win:SID"

### ISSUE THREAD

**Issue by gabiganam** [microsoft/perfview#1005]
In Windows 10 Update 1903, the "Microsoft-Windows-Kernel-Process" manifest was updated, and a new type "win:SID" was introduced.
This type is of variable length, it is a simple security identifier.

ProviderManifest.GetTypeForManifestTypeName returns null for "win:SID", and thus the parser doesn't parse anything after that field (so it misses the most important "ImageName" field).

**Comment by gabiganam** [comment]
Nobody? Nothing?
:(

**Comment by brianrob** [comment]
@gabiganam, you are correct that we don't have support for this opcode.  Let's keep this issue around to track this.

**Comment by gabiganam** [comment]
@brianrob thanks for your feedback Brian.
Please take a look at the additional (related) issue I opened: https://github.com/microsoft/perfview/issues/1004
